package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "css-audit": "node ./index.js",
     "lint:js": "eslint src",
-    "format:js": "prettier src *.js --write",
+    "format:js": "prettier src/**/*.js *.js --write",
```

Comment:
I added this because it was processing the CSS file – we can do that, if it seems better.